### ISSUE TITLE
Windows PHP 8.0 version old

### ISSUE THREAD

**Issue by GrahamCampbell** [shivammathur/test-setup-php#1]
Is there any chance we could get the 8.0.0 alpha3 release available on Windows? The current version is a bit old: https://github.com/shivammathur/test-setup-php/runs/893897861?check_suite_focus=true. :)

**Comment by shivammathur** [comment]
Yes, if till next release there aren't new official snap builds for Windows, I will create a nightly builder for Windows similar to [php-builder](https://github.com/shivammathur/php-builder).

**Comment by GrahamCampbell** [comment]
I think the snaps will be available on Thursday.
